code/game/vehicles/spacepods/equipment.dm

```diff
@@ -186,3 +187,42 @@
 /obj/item/device/spacepod_equipment/cargo/ore/removed(var/mob/user)
 	. = ..()
 	unload()
+
+/obj/item/device/spacepod_equipment/lock
+	name = "pod lock"
+	desc = "You shouldn't be seeing this"
+	icon = 'icons/goonstation/pods/ship.dmi'
+	icon_state = "blank"
+	var/mode = 0
+	var/id = null
+
+// Key and Tumbler System
+/obj/item/device/spacepod_equipment/lock/keyed
+	name = "\improper spacepod tumbler lock"
+	desc = "A locking system to stop podjacking. This version uses a standalone key."
+	icon_state = "pod_locator"
+	var/static/id_source = 0
```

Comment:
```/static``` is <i>required</i> here for this method of unique id generation. (```/global``` also works but using that keyword is misleading)